src/planscape/plan/views.py

```diff
@@ -333,7 +311,8 @@ def get_project_areas(request: HttpRequest) -> HttpResponse:
 def get_scores(request: HttpRequest) -> HttpResponse:
     try:
         with connection.cursor() as cursor:
-            plan = get_plan_by_id(request.GET)[0]
+            user = _get_user(request)
+            plan = get_plan_by_id(user, request.GET)
             geo = plan.geometry
             reg = plan.region_name.removeprefix('RegionName.').lower()
 
```

Comment:
Can you check the next lines?  I see type errors in line 320 and 323, which I think can be corrected by changing the lines to

  c.pk: c.condition_name

and 

  c.raster_name: c.condition_dataset.pk